src/Tabs/TabsWrapper.php

```diff
@@ -206,7 +206,7 @@ function twAddFrameTab(tabsid, label, url) {
       temp = panelId;
       return panelId;
     } else {
-      asyncAlertMsg('Fee Sheet tab is already opened', 5000, type = 'danger', size = '') ;
+      asyncAlertMsg('Fee Sheet tab is already opened', 3000, type = 'warning', size = '') ;
```

Comment:
This needs to be translated. Check me example but
`asyncAlertMsg(<?php echo xlj('Fee Sheet tab is already opened') ?>, 3000, type = 'warning', size = '')`

Reply:
also, type = 'warning should be just 'warning'

Reply:
> This needs to be translated. Check me example but
> `asyncAlertMsg(<?php echo xlj('Fee Sheet tab is already opened') ?>, 3000, type = 'warning', size = '')`

I tried that earlier but it was causing an error `unexpected token <'

Reply:
that xlj command looks good. should work fine :)
(note there are not quotes around it)

Reply:
@bradymiller Yes there is no quotes around it
![Screenshot (142)](https://user-images.githubusercontent.com/55142272/79276044-ea6e2c00-7ec4-11ea-915a-4d66c5cb050b.png)

Reply:
The Javascript seems to surround in EOD, If I correct, You can follow this PR and this https://github.com/openemr/openemr/blob/master/src/Tabs/TabsWrapper.php

Reply:
Thanks @stu01509 Working fine now

Reply:
![](https://1.bp.blogspot.com/-NX1EYU1K2wU/WDfdWHEHuyI/AAAAAAAD0b0/FPzQKzCTcasqiSLFLtaxfOu09kNpNFGhQCLcB/s1600/AS000934_01.gif)